crud/tarantool_test.go

```diff
@@ -87,6 +87,11 @@ var opObjManyOpts = crud.OperationObjectManyOpts{
 	Timeout: crud.MakeOptFloat64(timeout),
```

Comment:
Please, update the `requrests_test.go` too.

Reply:
Updated